src/react-image-lightbox.js

```diff
@@ -1476,7 +1477,8 @@ class ReactImageLightbox extends Component {
             ? global.window.document.body
             : undefined
         }
-        overlayClassName="modalOverlay"
+        role="dialog"
+        aria-labelledby='"DialogName'
```

Comment:
This looks off, shouldn't it be like this?

```suggestion
        aria-labelledby="DialogName"
```